releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.1 - April 02, 2020
+* Improved refresh for feature flags to reduce the number of calls made to App Configuration if no change is detected.
+* Fixed the issue that caused `TryRefreshAsync` to throw an `Azure.Identity.AuthenticationFailedException` when the `TokenCredential` used to fetch a key vault reference failed to authenticate.
+* Fixed the following issues when `optional` parameter is set to `true` in the method `AddAzureAppConfiguration`
+    * An exception might be thrown if the configuration store could not be accessed.
+    * `RefreshAsync` might ignore exceptions if configuration fails to load after a change is detected in a key with `refreshAll: true`.
+    * `TryRefreshAsync` would throw a `NullReferenceException` if the initial attempt to load the configuration in `IConfiguration.Build` failed.
+
```

Comment:
Being able to recover from an initial load-failure is a great feature. Please do have a separate bullet point for it and explain how to benefit from it.